tests/data/profile_stability/rhel8/stig.profile

```diff
@@ -185,6 +185,7 @@ selections:
 - partition_for_var
 - partition_for_var_log
 - partition_for_var_log_audit
+- partition_for_var_tmp
```

Comment:
RHEL-8 STIG profile also includes rules for mount options for `/var/tmp/`, so this change should be good for now.
CC: @ggbecker and @carlosmmatos for awareness.

Reply:
Despite RHEL8 STIG having the mount options for `/var/tmp/`, the V1R1 doesn't mention this as a separate partition. Which I believe is a bug in the STIG. I guess since they have based the STIG on OSPP, that might explain why they miss the separate partition there. We have to bring this topic to the table. Whatever gets into this STIG profile that is extending OSPP will get overridden by the new standalone profile. So I don't have a strong opinion here.